server/src/main/java/com/sauron/service/TransactionServiceImpl.java

```diff
@@ -31,9 +38,9 @@ public TransactionDto getTransaction(long id) {
 
     @Override
     public Collection<TransactionDto> getAllTransactions() {
-        return transactionRepository.findAll().stream()
-                .map(this::convertToDto)
-                .collect(Collectors.toList());
+        Collection<BankRequest<Collection<TransactionDto>>> getAllTransactions = bankRequestService.getAllRequests();
+        Collection<Collection<TransactionDto>> dummyBankTransactionsResponse = bankServiceExecutor.execMethod(getAllTransactions);
```

Comment:
You could inline the variable to avoid such an ugly declaration.

Reply:
Good point. Fixed.